apps/settings/elements/accessibility.html

```diff
@@ -13,6 +13,12 @@ <h1 data-l10n-id="accessibility-header"></h1>
             <small id="colorFilter-desc">Disabled</small>
           </a>
         </li>
+        <li>
+          <a class="menu-item" href="#accessibility-monoaudio">
+            <span data-l10n-id="monoAudio">Mono Audio</span>
```

Comment:
And hence lets change monoAudio to audio where appropriate (everywhere except the actual setting switch)